### PR TITLE
Ios10 experimental

### PR DESCRIPTION
**Pull request by yoshisuga**
Fixed to get RetroArch to run on iOS 10, but only supports statically linked cores

### CODE REVIEW

File: command.c [hunk 1]
```diff
@@ -1794,7 +1794,7 @@ bool command_event(enum event_command cmd, void *data)
       case CMD_EVENT_LOAD_CORE:
          command_event(CMD_EVENT_LOAD_CORE_PERSIST, NULL);
 #ifndef HAVE_DYNAMIC
-         command_event(CMD_EVENT_QUIT, NULL);
+         // command_event(CMD_EVENT_QUIT, NULL);
 #endif
```
Comment (Alcaro) [inline command.c]: uh? Are you sure?

File: pkg/apple/iOS/Info.plist [hunk 2]
```diff
@@ -25,11 +25,11 @@
 	<key>CFBundleIcons~ipad</key>
 	<dict/>
 	<key>CFBundleIdentifier</key>
-	<string>com.libretro.RetroArch</string>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
 	<key>CFBundleInfoDictionaryVersion</key>
 	<string>6.0</string>
 	<key>CFBundleName</key>
-	<string>RetroArch</string>
+	<string>RetroArchTest</string>
```
Comment (Alcaro) [inline pkg/apple/iOS/Info.plist]: I don't think this one is intentional either.

File: command.c (hunk 1 above)
Comment (yoshisuga) [inline command.c]: Nope! I fat fingered a delete and I thought I committed a change that undid this but I must not have pushed this. In any case, this PR isn't meant to be merged I guess and I need to come up with a better way to do this just for iOS 10

File: pkg/apple/iOS/Info.plist (hunk 2 above)
Comment (yoshisuga) [inline pkg/apple/iOS/Info.plist]: Nope..:( this PR was just so twinaphex can see my changes

**Comment by yoshisuga** [comment]
Lets abandon this PR.

**Pull request closed without merging**